testsuite/E72.py

```diff
@@ -80,3 +78,8 @@ def func_histype(a, b, c):
     pass
 except Exception:
     pass
+#: Okay
+from . import custom_types as types
+
```

Comment:
These test files aren't ever executed so we don't need a real module backing them here

Reply:
thanks, removed `custom_types.py` in 5df259b